packages/lib/client-only/download-pdf.ts

```diff
@@ -0,0 +1,26 @@
+import type { DocumentData } from '@documenso/prisma/client';
+
+import { getFile } from '../universal/upload/get-file';
+
+type DownloadPDFProps = {
+  documentData: DocumentData;
+  fileName?: string;
+};
+
+export const downloadPDF = async ({ documentData, fileName }: DownloadPDFProps) => {
+  const bytes = await getFile(documentData);
+
+  const blob = new Blob([bytes], {
+    type: 'application/pdf',
+  });
+
+  const link = window.document.createElement('a');
+  const baseTitle = fileName?.includes('.pdf') ? fileName.split('.pdf')[0] : fileName;
```

Comment:
`const [baseTitle] = fileName?.includes('.pdf') : fileName.split('.pdf') : [fileName ?? 'document'];`

This way you can do `${baseTitle}_signed.pdf` without needing the ternary below 😄